hugo/content/tools/_index.md

```diff
@@ -63,6 +99,40 @@ sudo apt install podman rootlesskit slirp4netns
 
 should be enough to get you a working Podman setup.
 
+{{< /tab >}}
+{{< tab "macOS" >}}
+Podman for macOS is distributed using brew.
+```
+brew install podman
+```
+
+Next, create and start your first Podman machine:
+
+```
+podman machine init
+podman machine start
```

Comment:
Do you have to do something special if you want to be able to run tkey-builder on an arm64 Mac? The image we distribute is amd64 so perhaps the Podman VM can be set up to be amd64 too?

Reply:
No, it will emulate for you. My experience is that it will default to whatever platform the image has, but since you did not explicitly say "amd64" during your podman run you will get a warning since it expects it to be the same as the host, but it works just fine. 

The correct way for us (and to remove the warning) would be to add either "--platform=linux/amd64" or "--arch=amd64" in all our podman targets in our makefiles.

I have a function in my .zshrc like this
```
podman-run (){
     podman run --arch=amd64 --rm --mount type=bind,source="$(pwd)",target=/src -w /src -it ghcr.io/tillitis/tkey-builder:2 "$@"
 }
```
and I do not receive any warnings.